packages/sdk-client/src/client/api-client-helpers.ts

```diff
@@ -51,10 +50,10 @@ export async function invalidateAndRegenerateJwt(
   errorContext: ErrorContext,
 ): Promise<RequestOptions> {
   const oauth2Plugin = requestPlugins?.find(
-    (plugin) => plugin instanceof Oauth2TokenRequest,
-  ) as Oauth2TokenRequest;
+    (plugin) => plugin.getName() === 'Oauth2TokenRequest',
```

Comment:
Can we imagine having constants shared by plugin to avoid hard coded string comparison?
(in the future in case any change from plugin's name; code will be broken)

Reply:
Sure thing! I've updated the code in a new commit